easybuild/framework/easyconfig/tools.py

```diff
@@ -282,7 +287,7 @@ def det_easyconfig_paths(orig_paths, from_pr=None, easyconfigs_pkg_paths=None):
                     break
 
                 # ignore subdirs specified to be ignored by replacing items in dirnames list used by os.walk
-                dirnames[:] = [d for d in dirnames if not d in build_option('ignore_dirs')]
+                dirnames[:] = [d for d in dirnames if d not in build_option('ignore_dirs')]
```

Comment:
@wpoely86: OK, I think from your part well done and said.

If anyone knows why pep8 triggered the warning, it would be nice to know; 
perhaps code maintainability as such an expression may perhaps grow later on...?